### ISSUE TITLE
Crash on upgrade

### ISSUE THREAD

**Issue by florianschanda** [pyanodon/pyalienlife#17]
I have updated from 1.8.0 to 1.8.1 and now my save seems bricked. This is triggered whenever I build *anything*.

```
The mod Pyanodons AlienLife (1.8.1) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event pyalienlife::on_built_entity (ID 6)
__pyalienlife__/control.lua:882: attempt to index field 'tech_upgrades' (a nil value)
stack traceback:
	__pyalienlife__/control.lua:882: in function <__pyalienlife__/control.lua:806>
```

Save attached.
[Flo_PY.zip](https://github.com/pyanodon/pyalienlife/files/4842639/Flo_PY.zip)

**Comment by SuperSandro2000** [comment]
The title is misleading.  A better one would be: Crash when building anything in 1.8.1

> now my save seems bricked

It is not. Just downgrade.

**Comment by florianschanda** [comment]
I mean, sure ;) I have done that it's fine for now. I just figured it would be upgrade related since then is when stuff normally breaks.

**Comment by kingarthur91** [comment]
this should be fixed with 1.8.2. closing it now
